account_mass_reconcile_transaction_ref/tests/test_account_reconcile_transaction_ref.py

```diff
@@ -72,6 +72,7 @@ def test_mass_reconcile_transaction_ref_vs_ref(self):
         })
         count = reconcile.unreconciled_count
         reconcile.run_reconcile()
+        reconcile._get_total_unrec()
```

Comment:
I'm not sure about this one, IMO it's an issue with the dependencies of `_get_total_unrec` : https://github.com/OCA/account-reconcile/blob/11.0/account_mass_reconcile/models/mass_reconcile.py#L119

Maybe we should add `history_ids` to make sure the field is recomputed after `run_reconcile` ?

Reply:
added